src/backend/common/helpers/insights_helper.py

```diff
@@ -290,20 +295,45 @@ def _create_leaderboard_from_dict_counts(
         )
 
     @classmethod
-    def _calculate_leaderboard_blue_banners(
+    def _calculate_leaderboard_most_awards_and_most_blue_banners(
         cls, award_futures: List[TypedFuture[Award]], year: Year
     ) -> List[Insight]:
-        data = defaultdict(int)
+        banner_count = defaultdict(int)
+        award_count = defaultdict(int)
         for award_future in award_futures:
             award = award_future.get_result()
-            if award.award_type_enum in BLUE_BANNER_AWARDS and award.count_banner:
+            for team_key in award.team_list:
+                award_count[team_key.id()] += 1
+                if award.award_type_enum in BLUE_BANNER_AWARDS and award.count_banner:
+                    banner_count[team_key.id()] += 1
+
+        return [
+            cls._create_leaderboard_from_dict_counts(
+                banner_count, Insight.TYPED_LEADERBOARD_BLUE_BANNERS, year
+            ),
+            cls._create_leaderboard_from_dict_counts(
+                award_count, Insight.TYPED_LEADERBOARD_MOST_AWARDS, year
+            ),
+        ]
+
+    @classmethod
+    def _calculate_leaderboard_most_regional_district_event_wins(
+        cls, award_futures: List[TypedFuture[Award]], year: Year
+    ) -> List[Insight]:
+        count = defaultdict(int)
+        for award_future in award_futures:
+            award = award_future.get_result()
+            if award.award_type_enum == AwardType.WINNER and award.event_type_enum in [
+                EventType.REGIONAL,
+                EventType.DISTRICT,
```

Comment:
Is this intentionally excluding district champs?

Reply:
Oh I thought the prod one excluded dcmp. Oops. will fix